""" exception when the hero finds an item """

class FoundItem(Exception):
    pass